components/measurement/nettests/web_connectivity.js

```diff
@@ -242,25 +244,34 @@ const WebConnectivityDetails = ({ measurement, render }) => {
                   {queries.map((query, index) => <QueryContainer key={index} query={query} />)}
                 </React.Fragment>
               } />
-              <DetailsBox pl={2} title='TCP Connections' content={
+              <DetailsBox pl={2} title={<FormattedMessage id='Measurement.Details.Websites.TCP.Heading' />} content={
                 <React.Fragment>
-                  {tcpConnections.length === 0 && <Text>No results</Text>}
+                  {tcpConnections.length === 0 && <Text><FormattedMessage id='Measurement.Details.Websites.TCP.NoData' /></Text>}
                   {tcpConnections.map((connection, index) => (
                     <Flex key={index}>
                       <Box>
-                        <Text>Connection to <strong>{connection.destination}</strong> was {connection.status}.</Text>
+                        <Text>
+                          <FormattedMessage
```

Comment:
This one has parameters and a bit of logic involved in it. [Here's the API reference](https://formatjs.io/guides/message-syntax/#select-format).
It would be great if @agrabeli can give feedback about how much of a problem this could be for translators.

Reply:
I think this may be a bit overwhelming for translators. Moreover it's a format.js specific, which means it will only work as a translation in JS based project and will not be easy to integrate into things like our apps.

I would suggest, even if that means a bit more LOC, to split this out into individual messages for each state.

Example:

```
{status == "successful" && <FormattedMessage id="Measurement.Details.Website.TCP.ConnectionTo.Successful" />}
{status == "failed" && <FormattedMessage id="Measurement.Details.Website.TCP.ConnectionTo.Failed" />}
{status == "blocked" && <FormattedMessage id="Measurement.Details.Website.TCP.ConnectionTo.Blocked" />}
```